site/src/patterns/accordion.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Accordion
+description: Accordion patterns in the Atlas Design System
+template: standard
+# figmaEmbed: https://www.figma.com/design/a5gqz0qtUMAKF36lNS6b61/Adaptive-Learning---Phase-I-(2024)?node-id=2581-106784&t=lkaa1re80XFTF9O3-0
+---
+
+# Accordion patterns
+
+Because the `details` component has so many possible permutations, it can be helpful to provide examples of certain styles.
```

Comment:
I wouldn't refer to details component but accordion instead. 

Because of the accordion component ....

Reply:
✅Changed it to be more in line with the card component's reference to the card patterns page.

Reply:
still see the `details` uses, please change to `accordion`

Reply:
This has been updated.